src/Exception/DispatchingException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Equip\Exception;
+
+use InvalidArgumentException;
+
+class DispatchingException extends InvalidArgumentException
+{
+    /**
+     * @param string $spec
+     *
+     * @return static
+     */
+    public static function notInvokable($spec)
+    {
+        return new static(sprintf(
+            'Dispatch class `%s` is not invokable',
```

Comment:
This seems misleading since it's possible that `$spec` may be a `callable` rather than a class instance.